js/nova.js

```diff
@@ -186,9 +188,18 @@ module.exports = class nova extends Exchange {
             'pair': market['id'],
         };
         let response = await this.privatePostTradePair (this.extend (order, params));
+        let orderid = undefined;
+        const tradeitems = this.safeValue (response, 'tradeitems');
+        if (tradeitems !== undefined) {
+            for (let i = 0; i < tradeitems.length; i++) {
+                if (this.safeString (tradeitems[i], 'type') === 'created') {
+                    orderid = this.safeString (tradeitems[i], 'orderid');
```

Comment:
> @kroitor i can do that once i have some balances on Nova. Don't have any at the moment

@cod3gen I see... The reason I'm asking is because the limit order doesn't necessarily have trades... I wonder how do they return the order id for a limit order that is still open and is not filled with any trades yet...

Reply:
Nevermind, I think I got it now, they return a special trade item when the order is opened.